### ISSUE TITLE
Repo changes

### ISSUE THREAD

**Issue by srinandan** [apigee/istio-mixer-adapter#105]
I think we should move the following files to a samples folder:

api-spec.yaml
apigee-handler.yaml
apigee-rule.yaml
authentication-policy.yaml

I would like us to include the helloworld.yaml in the samples folder.

In the install folder, we should have:
apigee-definitions.yaml
apigee-istio-demo.yaml
apigee-istio-demo-auth.yaml

The last two yaml files contain the change to the mixer image.

**Comment by theganyo** [comment]
I don't think we should be maintaining a version of the istio install files.

**Comment by srinandan** [comment]
I agree with you. Once the adapter is available over gRPC, we would not have to. Until then, for two reasons it makes sense: 1) a version of istio we know the adapter works with. a new user typically installs the latest version of istio and that may not work with the adapter. 2) convenience & user experience. it feels "hacky" to edit a file manually before installation.

The Istio team now calls the install file a demo (which makes sense). This would be our demo file. I'm happy if moved those files to the samples folder.

**Comment by theganyo** [comment]
Ok. I'm just going to get rid of the `install` directory entirely like Istio did. I'll rename `install` to `samples` and I'll add the necessary files there.
